Lib/KeychainAccess/Keychain.swift

```diff
@@ -1361,6 +1370,12 @@ extension Options {
             }
         }
         #endif
+        
+        if #available(iOS 13.0, OSX 10.15, watchOS 6.0, tvOS 13.0, *) {
+            if accessGroup != nil {
+                query[UseDataProtectionKeychain] = true
+            }
```

Comment:
Automatically set `UseDataProtectionKeychain` to `true` when an access group was set. This is *required* for macOS and doesn't affect other platforms